### ISSUE TITLE
Support reverse mode

### ISSUE THREAD

**Issue by Himura2la** [edgard/iperf3_exporter#8]
This feature is already implemented in the code. Reverse mode is supported via the `reverse_mode` query parameter which can be set to `true` to enable the `-R` flag in iperf3.

**Comment by cooperlees** [comment]
Isn't this `iperf3_received_bytes` ?

**Comment by Himura2la** [comment]
It's about adding the ability to pass the `-R` flag to iperf3. 
I guess it would require adding а relabel_config like `__reverse_mode__` with values `true/false` with default `false`

**Comment by cooperlees** [comment]
Yeah, I was just about to reply. I went and read the code, I see it's NOT. I'd like this too. I might look at trying to do a PR. But I'm a go n00b, but should be able to copy pasta enough and guess.

**Comment by Himura2la** [comment]
Cool, go on, I can test this ))

**Comment by edgard** [comment]
Restoring original issue content. The previous edit was made in error.
